invenio_communities/assets/semantic-ui/js/invenio_communities/subcommunity/search.js

```diff
@@ -6,47 +6,49 @@
  * under the terms of the MIT License; see LICENSE file for more details.
  */
 
-import { createSearchAppInit } from "@js/invenio_search_ui";
+import React from "react";
+import ReactDOM from "react-dom";
 import {
   ContribBucketAggregationElement,
   ContribBucketAggregationValuesElement,
   ContribSearchAppFacets,
 } from "@js/invenio_search_ui/components";
-import { overrideStore, parametrize } from "react-overridable";
+import { parametrize } from "react-overridable";
 import {
-  CommunitiesResults,
   CommunitiesSearchBarElement,
-  CommunitiesSearchLayout,
   CommunityItem,
   ResultsGridItemTemplate,
+  CommunitySelectionSearch,
 } from "../community";
 
-const appName = "InvenioSubCommunities.Search";
+import { Container } from "semantic-ui-react";
+import { SubcommunityResults } from "./SubcommunityResults";
 
 const ContribSearchAppFacetsWithConfig = parametrize(ContribSearchAppFacets, {
   help: false,
 });
 
-const CommunitiesSearchLayoutConfig = parametrize(CommunitiesSearchLayout, {
-  appName: appName,
-});
-
-export const defaultComponents = {
-  [`${appName}.BucketAggregation.element`]: ContribBucketAggregationElement,
-  [`${appName}.BucketAggregationValues.element`]: ContribBucketAggregationValuesElement,
-  [`${appName}.SearchApp.facets`]: ContribSearchAppFacetsWithConfig,
-  [`${appName}.ResultsList.item`]: CommunityItem,
-  [`${appName}.ResultsGrid.item`]: ResultsGridItemTemplate,
-  [`${appName}.SearchApp.layout`]: CommunitiesSearchLayoutConfig,
-  [`${appName}.SearchBar.element`]: CommunitiesSearchBarElement,
-  [`${appName}.SearchApp.results`]: CommunitiesResults,
+export const overriddenComponents = {
+  [`BucketAggregation.element`]: ContribBucketAggregationElement,
+  [`BucketAggregationValues.element`]: ContribBucketAggregationValuesElement,
+  [`SearchApp.facets`]: ContribSearchAppFacetsWithConfig,
+  [`ResultsList.item`]: CommunityItem,
+  [`ResultsGrid.item`]: ResultsGridItemTemplate,
+  [`SearchBar.element`]: CommunitiesSearchBarElement,
+  [`SearchApp.results`]: SubcommunityResults,
```

Comment:
Does it make sense to have the overridden components defined outside the element? Since this is now a "dedicated" search component for subcommunities, perhaps this could be defined in the component itself

Reply:
If we move the component to react-invenio-forms, we should use props rather than overrideable.

Reply:
I am not sure I understood what using `props` means in this context. But let's have a discussion IRL with @ntarocco and @kpsherva  to align